src/Tizen.NUI.Components/Theme/DefaultThemeCommon.cs

```diff
@@ -420,8 +420,8 @@ public Theme Create()
             // AppBar base style
             theme.AddStyleWithoutClone("Tizen.NUI.Components.AppBar", new AppBarStyle()
             {
-                Size = new Size(-1, 120),
-                BackgroundColor = new Color("#EEEFF1"),
+                Size = new Size(-1, 64),
+                BackgroundColor = new Color(0, 0, 0, 0),
```

Comment:
How about to use Color.Transparent ?

Reply:
Thank you for your comment! I have updated to use Color.Transparent.